x/crosschain/keeper/evm_hooks.go

```diff
@@ -72,12 +74,25 @@ func (k Keeper) ProcessLogs(ctx sdk.Context, logs []*ethtypes.Log, emittingContr
 				return err
 			}
 		}
+		// We were able to parse the ZRC20 withdrawal event. However, we were unable to process it as the information was incorrect
+		// This means that there are some funds locked in the contract which cannot have an outbound , this can be a candidate for a refund
+		// TODO : Consider returning error or auto refunding the funds to the user
+		if eventWithdrawal != nil {
```

Comment:
shouldn't it be an `else if eventWithdrawal != nil {` ?

Reply:
aaea061766d4b155b32783fcb417ae9388328e95